specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -36,13 +36,59 @@ model AcsUserDisconnectedEventData {
   userCommunicationIdentifier: CommunicationIdentifierModel;
 }
 
+/** Schema of the Data property of an EventGridEvent for a Microsoft.Communication.CallStarted event. */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallStartedEventData
+  extends AcsCallingEventProperties {
+}
+
+/** Schema of the Data property of an EventGridEvent for a Microsoft.Communication.CallEnded event. */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+model AcsCallEndedEventData
+  extends AcsCallingEventProperties {
+  /** The communication identifier of the user who was disconnected */
+  endedBy?: AcsCallEndedByProperties;
+
+  /** The reason for ending the call. */
+  reason?: AcsCallEndReasonProperties;
+
+  /** Duration of the call in seconds. */
+  callDurationInSeconds?: float64;
```

Comment:
This should be `Duration`?

Reply:
I think we will need to also use the `@encode` decorator so that it still gets encoded as a float.